src/mercury_engine_data_structures/formats/bcskla.py

```diff
@@ -22,50 +20,47 @@
     this,
 )
 
-from mercury_engine_data_structures.adapters.enum_adapter import EnumAdapter
 from mercury_engine_data_structures.common_types import Float, VersionAdapter
 from mercury_engine_data_structures.construct_extensions.alignment import AlignTo
 from mercury_engine_data_structures.formats.base_resource import BaseResource
 from mercury_engine_data_structures.formats.property_enum import PropertyEnumDoubleUnsafe
 from mercury_engine_data_structures.game_check import Game
 
 
-class TimingTypeEnum(Enum):
-    ONE_BYTE = 8
-    TWO_BYTE = 0
-
-
 class DreadKFVAdapter(Adapter):
+    class TimingType:
```

Comment:
should this derive from enum.Enum?

Reply:
I actually wanted just numbers in a namespace